prog/sensors/chips.c

```diff
@@ -105,12 +105,12 @@ void print_chip_json(const sensors_chip_name *name)
 					if (subCnt > 0)
 						printf(",\n");
 					printf("         \"%s\": %.3f", sub->name, val);
+					subCnt++;
 				}
 
 			} else {
 				printf("(%s)", label);
```

Comment:
What about this branch?

Reply:
No, I meant there should be subCnt++ on line 112 too.

Reply:
Added it :heart_eyes: